pycqed/analysis_v2/tomography_V2.py

```diff
@@ -0,0 +1,553 @@
+        'measurement_basis' + 'measurement_basis_labels'
+        to values corresponding to your experiment
+        and maybe 'readout_basis'
+    """
+
+    # The set of single qubit rotation matrixes used in the tomography
+    # measurement (will be assumed to be used on all qubits)
+    rotation_matrixes = [qt.identity(2), qt.sigmax(),
+                         qt.rotation(
+                             qt.sigmay(), np.pi / 2), qt.rotation(qt.sigmay(), -1*np.pi / 2),
+                         qt.rotation(qt.sigmax(), np.pi / 2), qt.rotation(qt.sigmax(), -np.pi / 2)]
+    measurement_operator_labels = ['I', 'X', 'y', '-y', 'x','-x']
+    #MAKE SURE THE LABELS CORRESPOND TO THE ROTATION MATRIXES DEFINED ABOVE
+
+    # The set of single qubit basis operators and labels (normalized)
+    measurement_basis = [
+        qt.identity(2) , qt.sigmaz() , qt.sigmax() , qt.sigmay() ]
+    measurement_basis_labels = ['I', 'Z', 'X', 'Y']
+    # The operators used in the readout basis on each qubit
+    readout_basis = [qt.identity(2) , qt.sigmaz() ]
+
+    def __init__(self, n_qubits=2, check_labels=False):
+        """
+        keyword arguments:
+        measurements_cal --- Should be an array of length 2 ** n_qubits
+        measurements_tomo --- Should be an array of length length(rotation_matrixes) ** n_qubits
+        n_qubits --- default(2) the amount of qubits present in the expirement
+        n_quadratures --- default(1(either I or Q)) The amount of complete measurement data sets. For example a combined IQ measurement has 2 measurement sets.
+        tomo_vars  : since this tomo does not have access to the original data, the vars should be given by
+                        tomo_var_i = 1 / N_i * np.var(M_i) where i stands for the data corresponding to rotation i. 
+
+        """
+        self.n_qubits = n_qubits
+        self.n_states = 2 ** n_qubits
+
+        # Generate the vectors of matrixes that correspond to all measurements,
+        # readout bases and rotations
+        self.basis_vector = self._calculate_matrix_set(
+            self.measurement_basis, n_qubits)
+        self.readout_vector = self._calculate_matrix_set(
+            self.readout_basis, n_qubits)
+
+        self.rotation_vector = self._calculate_matrix_set(
+            self.rotation_matrixes, n_qubits)
+
+        # generate the basis change matrix from pauli to comp and back
+        A = np.zeros((self.n_states**2, self.n_states**2), dtype=complex)
+        for i in range(self.n_states**2):
+            # do an orthonormal transformation. 
+            A[:,i] = np.ravel(self.basis_vector[i].full()) 
+        self.basis_pauli_to_comp_trafo_matrix= A / self.n_states
+        self.basis_comp_to_pauli_trafo_matrix= np.linalg.inv(A) 
+
+        #get dims of qutip objects
+        self.qt_dims = [[2 for i in range(self.n_qubits)], [2 for i in range(self.n_qubits)]]
+
+        if check_labels is True:
+            print(self.get_meas_operator_labels(n_qubits))
```

Comment:
Unclear what it is that it prints, I assume the label should be equal to something else. Maybe add some comments or test to the print statement.